ward/_config.py

```diff
@@ -27,7 +27,7 @@ def read_config_toml(project_root: Path, config_file: str) -> _ConfigDict:
         return {}
 
     try:
-        pyproject_toml = tomli.loads(path.read_text(encoding="utf-8"))
+        pyproject_toml = tomli.loads(path.read_bytes().decode())
```

Comment:
This change disables newline normalization which should not be done when reading TOML. See e.g. https://github.com/hukkin/tomli/issues/99#issuecomment-885273176